flipperdevices/flipperzero-firmware
# [FL-2700] Fix IR hangup with short signals

# What's new

- Fixed IR application hang up when sending very short signals continuously.
- Ported in-app blinking notifications to hardware blinker.
- Removed duplicate notification messages across other apps.

# Verification 

- Save a test signal ([see bug attachment](https://flipperzero.atlassian.net/browse/FL-2700)) onto your Flipper.
- Make sure Flipper is connected to computer via USB.
- Transmit the signal for at least 5 seconds continuously. No hangups should occur.
- Use all modes that have a blinking notification. Blinking should work properly every time.

# Checklist (For Reviewer)
